ETWAnalyzer_uTest/EventDump/DumpProcessesTests.cs

```diff
@@ -194,6 +194,68 @@ public void SessionFilter_Matches_Numbers_Exactly()
             Assert.False(dumper.SessionIdFilter(procSession2));
         }
 
+        [Fact]
+        public void User_Filter_Null()
+        {
+            DumpProcesses dumper = new DumpProcesses()
+            {
+                User = Matcher.CreateMatcher(null),
+            };
+
+            var procUser1 = new ETWProcess
+            {
+                Identity = null
+            };
+
+            Assert.True(dumper.UserFilter(procUser1));
+        }
+
+        [Fact]
+        public void User_Filter()
+        {
+            DumpProcesses dumper = new DumpProcesses()
+            {
+                User = Matcher.CreateMatcher("*SERVICE*"),
+            };
+
+            var procUser1 = new ETWProcess
```

Comment:
the naming could be improved. Like procSystem, procLocalService, procNetworkService, .... Then one can read the expected asserts better.

Reply:
resolved